test/test-restart-all.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+var _ = require('lodash');
+var cluster = require('cluster');
+var debug = require('debug')('strong-cluster-control:test');
+var master = require('../lib/master');
+var tap = require('tap');
+
+debug('master', process.pid);
+
+cluster.setupMaster({
+  exec: 'test/workers/null.js'
+});
+
+tap.test('restart all workers', function(t) {
+  var SIZE = 5;
+  var DELAY = 200;
+
+  t.plan(4);
```

Comment:
It needs a plan. It has two events, the test isn't over until they are both emitted. If you didn't have a plan, put the t.end() before the master.stop, and you deleted the code that emits the 'startRestart' event, then the test would pass, but without ever having seen that event, or done the deepEqual on its argument.